docs/adr/ADR-004a-multi-hypervisor-implementation-details.md

```
@@ -0,0 +1,837 @@
+- ❌ Not supported
+
+---
+
+## 2. Resource Hierarchy Mapping
+
+Each hypervisor has a different resource organization model. We need a translation layer.
+
+### 2.1 Resource Hierarchy Models
+
+**VMware vSphere:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)